pkg/auth/authz_handler_test.go

```diff
@@ -818,30 +784,35 @@ func postStepReply(ctx context.Context, cli *client.AssistedInstall) error {
 }
 
 func uploadHostLogs(ctx context.Context, cli *client.AssistedInstall) error {
+	hostId := strfmt.UUID(uuid.New().String())
 	file, err := ioutil.TempFile("/tmp", "test.log")
 	if err != nil {
 		return err
 	}
-	_, err = cli.Installer.UploadHostLogs(
+	_, err = cli.Installer.V2UploadLogs(
 		ctx,
-		&installer.UploadHostLogsParams{
+		&installer.V2UploadLogsParams{
 			ClusterID: strfmt.UUID(uuid.New().String()),
-			HostID:    strfmt.UUID(uuid.New().String()),
+			HostID:    &hostId,
 			Upfile:    file,
```

Comment:
I know you're not changing this in this PR, but this also doesn't need an actual file.